tests/ai_ml/tflite2.pm

```diff
@@ -28,8 +30,16 @@ sub run {
     # Extract model and labels
     assert_script_run('unzip ~/data/ai_ml/models/mobilenet_v1_1.0_224_quant_and_labels.zip');
 
-    # Run the test
-    assert_script_run("python3 ~/data/ai_ml/label_image.py --image ~/data/ai_ml/images/White_shark.jpg --model_file mobilenet_v1_1.0_224_quant.tflite --label_file labels_mobilenet_quant_v1_224.txt | tee /dev/$serialdev | head -n1", sub { m/great white shark$/ });
+    my $result = script_output("python3 ~/data/ai_ml/label_image.py --image ~/data/ai_ml/images/White_shark.jpg --model_file mobilenet_v1_1.0_224_quant.tflite --label_file labels_mobilenet_quant_v1_224.txt | tee /dev/$serialdev | head -n1", proceed_on_failure => 1);
+    record_info("TEST LOG", "$result");
+
+    if ($result !~ /great white shark/) {
+        if ($ret == 4 && is_opensuse) {
+            record_soft_failure("boo#1199429 nothing provides 'libprotobuf.so.30()(64bit)' needed by the to be installed tensorflow2");
+        } elsif ($ret == 104 && is_leap) {
+            record_soft_failure("boo#1199330 package python3-tensorflow is not found");
+        }
```

Comment:
Looks nice! Just a minor note that caught my attention, if the result does not match `great white shark` and the package installation proceeded fine, shouldn't the test die?

Reply:
Typically yes,  Test should fail if the python script(label_image.py) output does not match to the `great white shark`.  

For now I have temporarily added softfail as the python script failed to run due to known bug  (`failed to install python3-tensorflow`)

Reply:
Thank you Martin for the feedback :)